ntbs-service/Models/ServiceDirectoryItemWrapper.cs

```diff
@@ -0,0 +1,40 @@
+﻿using ntbs_service.Models.Entities;
+using ntbs_service.Models.ReferenceEntities;
+
+namespace ntbs_service.Models;
+
+public class ServiceDirectoryItemWrapper
+{
+    public User User { get; }
+    public PHEC Region { get; }
+    public TBService TBService { get; }
+    public Hospital Hospital { get; }
+
+    public ServiceDirectoryItemWrapper(User user = null, PHEC region = null, TBService tbService = null, Hospital hospital = null)
+    {
+        this.User = user;
+        this.Region = region;
+        this.TBService = tbService;
+        this.Hospital = hospital;
+    }
+
+    public bool IsUser()
```

Comment:
I think these would be better as get only properties, e.g.
```
public bool IsUser => User is not null;
```

Reply:
There's also an argument to say that these properties aren't needed at all, as calling classes have access to the `User` property directly, so could check whether it's null themselves. Encapsulating it in this property doesn't save much.